src/components/ImageDetails/VPhotoDetails.vue

```diff
@@ -29,23 +29,29 @@
         {{ $t('photo-details.legal-disclaimer') }}
       </p>
 
-      <VButton
-        variant="action-button"
-        size="disabled"
-        class="report py-2 mt-2"
-        @click="reportForm.toggleVisibility"
-      >
-        {{ $t('photo-details.content-report.title') }}
-        <VIcon :icon-path="icons.flag" class="text-trans-blue ms-2 text-sm" />
-      </VButton>
-      <VContentReportForm
-        v-if="isReportFormVisible"
-        :image="image"
-        :provider-name="providerName"
-        data-testid="content-report-form"
-        class="mt-2 text-left"
-        @close-form="reportForm.close()"
-      />
+      <VPopover z-pos="20">
```

Comment:
Need the `:` otherwise Vue will treat the value as a string. We also need a label for the popover content to be accessible 🙂 

```suggestion
      <VPopover :z-pos="20" :label="$t('photo-details.content-report.title')">
```

Reply:
Fixed by @obulat in #722 (merged in aeebb3f).